pkg/apis/core/validation/validation.go

```diff
@@ -4139,13 +4139,15 @@ func ValidatePodSpec(spec *core.PodSpec, podMeta *metav1.ObjectMeta, fldPath *fi
 		gracePeriod = *spec.TerminationGracePeriodSeconds
 	}
 
+	inUserNS := spec.SecurityContext != nil && spec.SecurityContext.HostUsers != nil && !*spec.SecurityContext.HostUsers
```

Comment:
nit: Consider inverting this and naming it `hostUsers`, to make it clear where the value comes from.

Reply:
updated!